src/custom-select/index.ts

```diff
@@ -17,41 +17,39 @@ class CustomSelect {
         this.$select.onSelect(event.item.i);
       }
       this.update({ isActive: false });
-      this.refs.toggle.blur();
     }
   };
   isActive: boolean = false;
 
+  init() {
+    utils.WINDOW.addEventListener('click', this.onClickDeactivate);
+  }
+
   onHoverActivate(event: MouseEvent & Tag.Event) {
     event.preventUpdate = true;
     if (this.props.hover && !this.isActive) {
       this.update({ isActive: true });
     }
   }
 
-  onClickDeactivate(event: MouseEvent & Tag.Event) {
+  onHoverDeactivate(event: MouseEvent & Tag.Event) {
     event.preventUpdate = true;
     if (!this.props.hover && !this.isActive) {
       this.update();
     }
   }
 
-  onLostFocus = (event: FocusEvent & Tag.Event) => {
+  onClickDeactivate = (event: MouseEvent & Tag.Event) => {
     event.preventUpdate = true;
-    if (!this.props.hover) {
-      this.isActive = false;
+    if (!this.props.hover && event.target !== this.refs.toggle.querySelector('button')) {
```

Comment:
In which case I would add a `tags` property typing of `tags: { 'gb-button': Button };` where `Button` is the default export of the `gb-button` component